content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info-clio.md

```diff
@@ -58,63 +51,202 @@ An example of a successful response:
+                        "duration_us": "1395156"
+                    },
+                    "channel_authorize": {
+                        "started": "1",
+                        "finished": "1",
+                        "errored": "0",
+                        "forwarded": "1",
+                        "duration_us": "2017"
+                    },
+                    "manifest": {
+                        "started": "1",
+                        "finished": "1",
+                        "errored": "0",
+                        "forwarded": "1",
+                        "duration_us": "1707"
+                    },
+                    "subscribe": {
+                        "started": "6",
+                        "finished": "6",
+                        "errored": "0",
+                        "forwarded": "0",
+                        "duration_us": "116"
+                    },
+                    "random": {
+                        "started": "1",
+                        "finished": "1",
+                        "errored": "0",
+                        "forwarded": "0",
+                        "duration_us": "111"
+                    },
+                    "ledger_data": {
+                        "started": "14",
+                        "finished": "3",
+                        "errored": "11",
+                        "forwarded": "0",
+                        "duration_us": "6179145"
+                    },
+                    "ripple_path_find": {
+                        "started": "1",
+                        "finished": "1",
+                        "errored": "0",
+                        "forwarded": "1",
+                        "duration_us": "1409563"
+                    },
+                    "account_channels": {
+                        "started": "14",
+                        "finished": "14",
+                        "errored": "0",
+                        "forwarded": "0",
+                        "duration_us": "1062692"
+                    },
+                    "submit": {
+                        "started": "6",
+                        "finished": "6",
+                        "errored": "0",
+                        "forwarded": "6",
+                        "duration_us": "11383"
+                    },
+                    "transaction_entry": {
+                        "started": "8",
+                        "finished": "5",
+                        "errored": "3",
+                        "forwarded": "0",
+                        "duration_us": "494131"
+                    }
+                },
+                "subscriptions": {
+                    "ledger": 0,
+                    "transactions": 0,
+                    "transactions_proposed": 0,
+                    "manifests": 2,
+                    "validations": 2,
+                    "account": 0,
+                    "accounts_proposed": 0,
+                    "books": 0
+                }
+            },
```

Comment:
I thought more about this, and I think these counters are likely to be moved to admin only sometime soon, so I think we should not document them